class/practice5.1/main.cpp

```diff
@@ -0,0 +1,59 @@
+#include <cassert>
+#include <iostream>
+#include "MegaData.h"
+
+void testMegaDataInitialization() {
+    MegaData data;
+    for (const auto& val : data.smallArray) {
+        assert(val == 42.0f);
+    }
+    for (const auto& val : data.bigArray) {
+        assert(val == 42.0);
+    }
+    std::cout << "testMegaDataInitialization passed." << std::endl;
+}
+
+void testMegaDataReset() {
```

Comment:
we can reuse our mini-testframework from previous lessons and std::println instad of cout.
also it would be great to have visual splitting of the logical blocks inside your test case(Build-Operate-Check or Arrange-Act-Assert) - for example, add empty lines between them